host/contracts/actions.go

```diff
@@ -178,6 +177,17 @@ func (cm *ContractManager) handleContractAction(id types.FileContractID, height
 		sp, err := cm.buildStorageProof(contract.Revision.ParentID, contract.Revision.Filesize, leafIndex, log.Named("buildStorageProof"))
 		if err != nil {
 			log.Error("failed to build storage proof", zap.Error(err))
+			cm.alerts.Register(alerts.Alert{
```

Comment:
Might be worthwhile to extract these into a helper to make sure they remain consistent apart from the "Message"